package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "jellyfin-chromecast",
+  "version": "3.0.0",
+  "description": "Cast receiver for Jellyfin",
+  "license": "GPL-2.0-or-later",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/jellyfin/jellyfin-chromecast.git"
+  },
+  "bugs": {
+    "url": "https://github.com/jellyfin/jellyfin-chromecast/issues"
+  },
+  "homepage": "https://jellyfin.org/",
+  "dependencies": {
+    "clean-webpack-plugin": "^3.0.0",
+    "copy-webpack-plugin": "^5.1.1",
+    "file-loader": "^6.0.0",
+    "webpack": "4.42.1",
+    "webpack-cli": "^3.3.11"
+  },
+  "scripts": {
+    "devBuild": "webpack --config webpack.config.js --mode=development",
+    "prodBuild": "webpack --config webpack.config.js --mode=production"
```

Comment:
I think these could be improved. NPM usually has some standard commands like `prepare` that serve an actual function but it would also be nice to pick one flavor and label it `build` as the default.

Reply:
That's a good idea. Maybe with a naming scheme similar to web?
`prepare` builds production.
`build:development`.
`build:production`.

Reply:
That sounds good to me.